azurelinuxagent/ga/monitor.py

```diff
@@ -58,25 +58,30 @@ def _operation_impl(self):
         #
         # Check the processes in the agent cgroup
         #
-        processes = CGroupConfigurator.get_instance().get_processes_in_agent_cgroup()
-
-        if processes is not None:
-            unexpected_processes = []
-
-            for (_, command_line) in processes:
-                if not CGroupConfigurator.is_agent_process(command_line):
-                    unexpected_processes.append(command_line)
-
-            # Report a small sample of unexpected processes
-            if len(unexpected_processes) > 0 and self._error_count < 5:
-                unexpected_processes.sort()
-                error = ustr(unexpected_processes)
-                if error != self._last_error:
-                    self._error_count += 1
-                    self._last_error = error
-                    message = "The agent's cgroup includes unexpected processes: {0}".format(error)
-                    logger.info(message)
-                    add_event(op=WALAEventOperation.CGroupsDebug, message=message)
+        processes_check_error = None
```

Comment:
any exception in the code to check processes should not prevent us from reporting metrics